src/test/isolation2/output/resgroup/resgroup_bypass.source

```diff
@@ -161,7 +161,7 @@ SELECT * FROM memory_result;
  rsgname        | ismaster | avg_mem 
 ----------------+----------+---------
  rg_bypass_test | 0        | 0.0     
- rg_bypass_test | 1        | 12.0    
+ rg_bypass_test | 1        | 24.0    
```

Comment:
Do we need to change these values, or we could use `resGroupPallocIgnoreStartup()` here? Or for lesser diff, leave the old name intact, and use a new name for the new allocation function?

Reply:
Why would we need this patch then? This diff shows that the startup memory is counted by all processes